queries.sql

```diff
@@ -7,4 +7,10 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight > 10.5;
```

Comment:
- [ ] In this project, you are missing this part of the requirements:

![image](https://user-images.githubusercontent.com/57421823/156368893-8b611384-ce07-43bf-bb35-c0ae057dd772.png)

## Explanation

This will help you to understand how transactions, commits, and rollbacks work.  It will also teach you how to update a table that has already been created.

## Hints

- A transaction starts with a `BEGIN` and ends with a `COMMIT` or `ROLLBACK` statement.
- According to the requirements, you will need to verify each commit, rollback, set, and delete statement with a `SELECT` statement to verify the state of the data after each step. Please, keep this in mind.

This is an example:

```SQL
BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```

Now that you know how to do it, please implement these requirements on your project 💪